src/tests/JIT/opt/perf/doublenegate/doublenegate.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Runtime.CompilerServices;
+
+namespace coreclr_test_13647
+{
+    class doublenegate
+    {
+        static void Main(string[] args)
+        {
+            Test1();
+            Test2();
+        }
```

Comment:
The test is expected to return 100 as a success.
Also this test doesn't actually assert if something goes wrong.

it'd be nice to add a test case when argument is not only a local variable.

Reply:
(that's why CI is red currently)

Reply:
Done, thanks!